mne/viz/backends/_pyvista.py

```diff
@@ -170,6 +170,7 @@ def __init__(self, fig=None, size=(600, 600), bgcolor='black',
         self.font_family = "arial"
         self.tube_n_sides = 20
         self.shape = shape
+        self.screenshot_filename = "screenshot.png"
```

Comment:
On Ubuntu the default filename is `~/Screenshot from 2020-12-14 11-03-55.png`, and on macOS it's `~/Desktop/Screen Shot 2021-...`. So my vote is for something like `~/MNE <date> <time>.png`.

Reply:
With actual space? :fearful:

Reply:
You can add underscores if it's too much for you :)